pages/linux/upower.md

```diff
@@ -0,0 +1,24 @@
+# upower
+
+> System utility to provide battery information and statistics.
+> More information: <https://upower.freedesktop.org/>.
+
+- Dump battery information:
+
+`upower --dump`
+
+- List connected battery-using devices:
+
+`upower --enumerate`
+
+- Listen for changes in battery status:
+
+`upower --monitor`
+
+- Watch for and print battery status changes:
```

Comment:
Since both examples are almost the same I would also phrase them the same.
```suggestion
- Watch for and print battery status changes:

`upower --monitor`

- Watch for and print detailed battery status changes:
```